templates/tags/SageFinancialProgramCodes.json

```diff
@@ -38,7 +38,7 @@
   "Mobile Toolbox Tech Core_112503",
   "Model AD-IU_116000",
   "NF - DoD w/UCF_122100",
-  "NF Analysis & Dissemination UCSF_121900",
+  "NF Analysis + Dissemination UCSF_121900",
```

Comment:
I realize this is picky but can we change NO PROGRAM_00000 to NO PROGRAM_000000, i.e. since all other codes are six digits, let's use six digits here too.

Reply:
done